src/dpl/include/dpl/Opendp.h

```diff
@@ -96,6 +96,15 @@ struct Master
   bool is_multi_row = false;
 };
 
+enum class Edge
+{
+  top,
+  bottom,
+  left,
+  right,
+  invalid
```

Comment:
Enums should be in all caps (TOP, BOTTOM), or if we're following the [Google style guide](https://google.github.io/styleguide/cppguide.html#Enumerator_Names) kTop, kBottom.